src/medCore/gui/settingsWidgets/medDatabaseSettingsWidget.cpp

```diff
@@ -84,11 +92,14 @@ void medDatabaseSettingsWidget::read()
 {
     // we always show the datalocation here, the medStorage class takes care of retrieving the correct one
     d->dbPath->setText(medStorage::dataLocation());
+    medSettingsManager * mnger = medSettingsManager::instance();
```

Comment:
Wouldn't two extra 'a's added to the variable name (mnger -> manager) really aid readability ? :P